notebooks/passenger_vehicle_co2_lca.py

```diff
@@ -0,0 +1,322 @@
+
+
+# Scenario 6: Köpa elcykel
+scenarios = {"6. Köpa elcykel": cars["A: Liten bensinbil"]}
+driven_distance = np.linspace(0, kilometers_at_change, 100)
+car_data_1 = co2analysis(scenarios, driven_distance, 0)
+
+scenarios = {"tmp1": cars["E: Elcykel"]}
+driven_distance = np.linspace(kilometers_at_change, 300000, 100)
+car_data_2 = co2analysis(scenarios, driven_distance, car_data_1[0][2][-1] * 1000)
+# ([key,driven_distance,C02_data/1000])
+
+car_data_1[0][1] = np.append(car_data_1[0][1], car_data_2[0][1])
+car_data_1[0][2] = np.append(car_data_1[0][2], car_data_2[0][2])
+car_data.append(car_data_1[0])
+
+# Scenario 7: Aldrig köpa bil
+scenarios = {
+    "7. Aldrig köpa bil": cars["D: Cykel"],
+}
+driven_distance = np.linspace(0, 300000, 100)
+car_data_1 = co2analysis(scenarios, driven_distance, 0)
+car_data.append(car_data_1[0])
+# -
+
+plt.figure(figsize=(10, 6), dpi=100)
+leg = []
+for value in car_data:
+    plt.plot(value[1], value[2])
+    leg.append(value[0])
+plt.legend(leg)
+plt.xlabel("kördistans [km]")
+plt.ylabel("CO2 [metric tonnes]")
+plt.show()
+
+# ## Från det personliga planet till världens fordonsflotta
+# Genom att tillgripa en naivistisk analys (vi ger inte någon rabatt då en bil säljs, vilket innebär att inköp av begagnad bil skulle ge en initial-CO2-kostnad av 0 ton och därmed räknar vi inte med avskrivning. Eftersom koldioxid ackumuleras i atmosfären är det dock i vår mening det rimligaste sättet att förhålla sig till problemet) har vi visat hur elektrifiering, i dagens SUV-tider, sällan innebär sänkta koldioxidutsläpp, och en övergång till 10% cykel *i kombination med inköp av elbil* endast är marginellt bättre.
+#
+# Vad har vi för siffror att förhålla oss till på samhällsnivå? I världen finns det grovt räknat två miljarder bilar. Hur gamla dessa är har jag svårt att hitta siffror för, men den omfattande svenska statistiken har siffror som vi kan tillgripa: I Sverige är en bil i genomsnitt 10år gammal, och skrotas i genomsnitt efter 18år.
+#
+# Låt oss vidare anta att de svenska förhållandena är representativa för världen i stort och att en bil körs 300000km i genomsnitt innan den skrotas (vilket är siffror som dyker upp vid googling - jag hittar inte någon siffra från scb eller trafa för att bekräfta detta för tillfället).
+#
+# Slutligen antar vi att bilarna rullar med konstant körsträcka varje år. Det innebär alltså att en tio år gammal genomsnittsbil har:
+#
+# $$
+# \frac{(18-10)}{18}*300000=133333\text{km}
+# $$
+#
+# kvar att köra innan den skrotas. Anta nu att de allra flesta bilar i världsflottan fortfarande är bensinbilar, och att de i genomsnitt har en förbrukning om 0,7l/mil. Vi kan då räkna ut hur många miljarder ton CO2 den befintliga fordonsflottan förväntas släppa ut innan den är förbrukad:
+
+# +
+average_remaining_distance = 133333  # km
+average_fuel_consumption = 0.07  # l / km
+total_number_of_cars = 2e9  # two billion cars world wide
+
+133000 * 0.07 * kgC02_per_litre * 2000000000 / 1000000 / 1000
```

Comment:
magic numbers

Reply:
Defined in the comment block above, but yes, should be named etc.